applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -3,13 +3,17 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
+#define _BSD_SOURCE
```

Comment:
What is this, and why is it above the #include?

Reply:
I'm using strlcpy, strlcat -- size-bounded string copying and concatenation function here.
The prototype in zephyr-sdk-0.14.2\arm-zephyr-eabi\arm-zephyr-eabi\include\string.h is defined by pre-processor __BSD_VISIBLE.
__BSD_VISIBLE is defined by pre-processor in zephyr-sdk-0.14.2\arm-zephyr-eabi\arm-zephyr-eabi\include\sys\features.h.

The #define _BSD_SOURCE should be before the first instance of #include string.h. Mind that string.h could be included by some other #include statements in your source file, so the easiest is to put the #define _BSD_SOURCE on the very first line of the source file.